addons/website/static/src/scss/website_visitor_views.scss

```diff
@@ -4,14 +4,21 @@
         .o_kanban_record {
             width: 100%;
             margin: 0px;
-            .o_kanban_grouped {
-                display:none !important;
-            }
         }
     }
     &.o_kanban_grouped {
-        .o_kanban_detail_ungrouped {
-            display:none !important;
+        .row > div {
+            flex: 1 1 100% !important;
+            max-width: 100% !important;
+            margin: 8px 0 !important;
+            text-align: left !important;
```

Comment:
Would add a rule on "text-lg-end" specifically, otherwise it's a bit unclear why you need to do this.